platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -112,15 +114,15 @@ private void addProvider() {
 
     private void init() {
         try {
-            keyStore = KeyStore.getInstance(System.getProperty("javax.net.ssl.keyStoreType"));
-            trustStore = KeyStore.getInstance(System.getProperty("javax.net.ssl.keyStoreType"));
```

Comment:
This would have been a bug in the future if we ever got to the point where we could actually run with the trust store and key store as different types. Right now we cannot have them be different types anyways.